README-pt.md

```diff
@@ -144,6 +144,15 @@ sudo launchctl load -w /Library/LaunchDaemons/limit.maxfiles.plist
 
 Esta solução funciona tanto para o MacOS Catalina quanto para o MacOS Mojave.
 
+### Troubleshooting for Out of Memory
+
+Se você executar o comando `make container-serve` e retornar o seguinte erro:
+```
+make: *** [container-serve] Error 137
+```
+
+Verifique a quantidade de memória disponível para o executor/runtime de container. No caso de Docker Desktop para macOS, abra o menu "Preferences..." -> "Resources..." e tente disponibilizar mais memória.
```

Comment:
Two corrections:
1. From: "executor/runtime de container"; To: "agente de execução de contêiner"
1. From: "No caso **de** Docker"; To: "No caso **do** Docker"

Reply:
Thanks @jailton ! Changes done, let me know if there's something pending. Cheers!